crates/egui_extras/Cargo.toml

```diff
@@ -55,6 +55,9 @@ puffin = ["dep:puffin", "egui/puffin"]
 ## Support loading svg images.
 svg = ["resvg"]
 
+## Support rendering text in svg images. Adds a few hundred kb to the binary.
```

Comment:
Does it work on web too?

Reply:
Looks like everything is fine except it cant find the fonts. Does egui bundle the raw font files it uses? if we could hand them to fontdb, that would work.

![image](https://github.com/emilk/egui/assets/6328589/cc794671-652d-47eb-b497-8e56f5e108b9)

Reply:
Yes, a few .ttf files are included with `include_bytes` in `crates/epaint/src/text/fonts.rs`

Reply:
Any chance we could expose an API to make those bytes available?

Reply:
They can be found in `ctx.fonts(|f| { f.lock().fonts.definitions().font_data }`

Reply:
Hmmm, that requires us having a valid `egui::Context` that has already been run however. Are you okay with me plumbing through context to this method (a public API change), or should we have another way to access the font data that doesn't need a `Context` ?